smartsim/_core/control/jobmanager.py

```diff
@@ -301,21 +306,25 @@ def restart_job(
             else:
                 self.jobs[entity_name] = job
 
-    def get_db_host_addresses(self) -> t.List[str]:
+    def get_db_host_addresses(self) -> t.Dict[str, t.List[str]]:
         """Retrieve the list of hosts for the database
 
-        :return: list of host ip addresses
-        :rtype: list[str]
-        """
+        :return: dictionary of db_identifiers for each database
```

Comment:
Consider making function description more precise, and reducing parameter description length